# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/webrtc/trunk/webrtc/modules/video_capture/android/java/src/org/webrtc/videoengine/CaptureCapabilityAndroid.java
# Commit: 55e2a4c5cab9
# Full Hash: 55e2a4c5cab961f0fa8627a490c4d714584b3065
# Author: Dan Minor <dminor@mozilla.com>
# Date: 2020-02-11 21:47:36
# Regressor Bug: 1578073
# File Overlap Count: 2
# Description:
#   Bug 1578073 - Move android video capture code to dom/media/systemservices; r=jib
#   
#   Although originally part of webrtc.org, this code has subsequently been
#   removed by upstream. Moving it to under dom/media should make it clearer that
#   this is code that we are maintaining and simplify future upstream merges.
# ==============================================================================

diff -r 27b45ca46f24 -r 55e2a4c5cab9 media/webrtc/trunk/webrtc/modules/video_capture/android/java/src/org/webrtc/videoengine/CaptureCapabilityAndroid.java
--- a/media/webrtc/trunk/webrtc/modules/video_capture/android/java/src/org/webrtc/videoengine/CaptureCapabilityAndroid.java	Mon Feb 10 17:07:25 2020 +0000
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,24 +0,0 @@
-/*
- *  Copyright (c) 2012 The WebRTC project authors. All Rights Reserved.
- *
- *  Use of this source code is governed by a BSD-style license
- *  that can be found in the LICENSE file in the root of the source
- *  tree. An additional intellectual property rights grant can be found
- *  in the file PATENTS.  All contributing project authors may
- *  be found in the AUTHORS file in the root of the source tree.
- */
-
-package org.webrtc.videoengine;
-
-import org.mozilla.gecko.annotation.WebRTCJNITarget;
-
-@WebRTCJNITarget
-public class CaptureCapabilityAndroid {
-    public String name;
-    public int width[];
-    public int height[];
-    public int minMilliFPS;
-    public int maxMilliFPS;
-    public boolean frontFacing;
-    public int orientation;
-}